pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
+    }
+
+    .microsite-content .can_button, .microsite-content #can_embed_form input[type="submit"], .microsite-content #can_embed_form .button, .microsite-content #donate_auto_modal input[type="submit"], .microsite-content #donate_auto_modal .button {
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}} !important;
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}} !important;
+      {% endif %}
+    }
+  </style>
+{% endblock %}
+
+<div class="microsite">
+
+{% block notification_banner %}
+<!-- TODO: conditional notification banner on page model -->
+{% endblock %}
+
+{% block nav %}
+  {% if page.standard_header_show %}
+    {{ block.super }}
+  {% endif %}
+{% endblock %}
 
 {% block content %}
-{% if page.secondary_content_show %}
-<div style="background-color: #{% firstof page.secondary_content_background_color 'FFFFFF' %}">
-	{{ page.secondary_content | richtext }}
-</div>
-{% endif %}
+  <div id="primary_content" class="container-fluid microsite-content">
+    {% if page.custom_header_show %}
+      <div class="row">
+        <nav id="custom_header" class="navbar navbar--microsite pa20">
+          <div class="navbar__upper container">
+            {% if page.facebook_url or page.twitter_url %}
+              <div class="navbar__social">
+                {% if page.facebook_url %}
+                  <a href="{{ page.facebook_url }}" target="_blank" class="social-icon--facebook" onclick="trackOutboundLink('{{ page.facebook_url }}', true);">Facebook</a>
+                {% endif %}
+
+                {% if page.twitter_url %}
+                  <a href="{{ page.twitter_url }}" target="_blank" class="social-icon--twitter" onclick="trackOutboundLink('{{ page.twitter_url }}', true);">Twitter</a>
+                {% endif %}
+              </div>
+            {% endif %}
+
+            {% if page.custom_header_image %}
+              {% image page.custom_header_image max-600x600 as custom_header_image %}
+              <img src="{{ custom_header_image.url }}" alt="{{page.title}}" class="microsite-header-image">
+            {% endif %}
+
+            {% if page.button_url and page.button_text %}
+              <div class="navbar__buttons">
+                <a id="primary-cta"
+                  href="{{ page.button_url }}"
+                  target="{{ page.button_url_new_window|yesno:'_blank,_self'}}"
+                  class="btn btn-primary"
+                  onclick="trackOutboundLink('{{ page.button_url }}', {{ page.button_url_new_window|yesno:'true,false'}});">
+                  {{ page.button_text }}
+                </a>
+              </div>
+            {% endif %}
+          </div>
+        </nav>
+      </div>
+    {% endif %}
+
+    <div class="container pt4 pb4">
+      <div class="col-md-6 mb20 fs18">
+        {{ page.primary_content | richtext }}
+      </div>
+
+      {% if page.primary_content_embed_code %}
```

Comment:
I thought this would affect layout above as well? Primary content should be more centered if there is no signup form on right side?

Reply:
Yeah should probably have this be conditional. Will make jira issue.

Reply:
Ok thanks, fine to do as follow up since current use case will have form.